Assets/Treasure/TDK-Private/Editor/TDKPackagerFull.cs

```diff
@@ -31,8 +29,9 @@ public static void ExportPackage()
                 "Assets/Treasure/TDK/Resources/TDKConnectThemeData.asset",
                 "Assets/Treasure/TDK/Runtime",
 
-                // code stripping
-                "Assets/link.xml"
+                // code stripping & compilation defines
+                "Assets/link.xml",
+                "Assets/csc.rsp"
```

Comment:
Updated my open PR to include this file as well when creating the release

Reply:
@lefarchi shoot, apologies for the overlap here